waku/node/peer_manager/peer_manager.nim

```diff
@@ -583,15 +584,16 @@ proc connectToRelayPeers*(pm: PeerManager) {.async.} =
   let totalRelayPeers = inRelayPeers.len + outRelayPeers.len
   let inPeersTarget = maxConnections - pm.outRelayPeersTarget
 
-  if inRelayPeers.len > pm.inRelayPeersTarget:
-    await pm.pruneInRelayConns(inRelayPeers.len - pm.inRelayPeersTarget)
+  # TODO: Temporally disabled. Might be causing connection issues
+  #if inRelayPeers.len > pm.inRelayPeersTarget:
+  #  await pm.pruneInRelayConns(inRelayPeers.len - pm.inRelayPeersTarget)
```

Comment:
I vote to remove that directly. If we need it in the future we can review this commit and easily restore it back. The TODO is likely to be there forever :)
```suggestion
```

Reply:
would like to leave it there, since its something I would like to revisit. Some peer prunning in inbound connections is needed, the open question is how.